src/main/webapp/profile.jsp

```diff
@@ -0,0 +1,218 @@
+<%@page import="java.sql.SQLException"%>
+<%@page import="java.sql.ResultSet"%>
+
+
+<!DOCTYPE html>
+
+<html>
+<head>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link
+  	href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+  	rel="stylesheet"
+  	integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+  	crossorigin="anonymous">
+  </head>
+  <title></title>
+</head>
+<body>
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">PROFILE PAGE</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+      <div>
+             <li class="nav-item">
+                <a class="btn btn-primary" href="Delete.jsp" role="button">Delete</a>
+               </li>
+               </div>
+              <br>
+              <div>
+              <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalCenter">
+                Update
+              </button>
+              </div>
+
+          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <li><a class="dropdown-item" href="#">Action</a></li>
+            <li><a class="dropdown-item" href="#">Another action</a></li>
+            <li><hr class="dropdown-divider"></li>
+            <li><a class="dropdown-item" href="#">Something else here</a></li>
+          </ul>
+        </li>
+
+      </ul>
+       <li class="nav-item">
+       <a class="btn btn-primary" href="LogoutServlet" role="button">Logout</a>
+       </li>
+      <form class="d-flex">
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+        <button class="btn btn-outline-success" type="submit" <a href="Search.jsp"></a>Search</button>
+      </form>
+    </div>
+  </div>
+</nav>
+<%
+String email=(String) session.getAttribute("email");
+String password = (String) session.getAttribute("password");
+String firstName="";
+String lastName="";
+String city="";
+String zip="";
+String state="";
+String country="";
+String phone="";
+String partyId="";
+
+     try {
+        Connection con =null;
+                 try{
+                     Class.forName("com.mysql.cj.jdbc.Driver");
+                     con = DriverManager.getConnection("jdbc:mysql://127.0.0.1:3306/adv", "priyesh", "123456");
```

Comment:
JDBC code should not be used in JSP file and don't use your credentials in the JSP file. Try to find alternative way of doing so.